management/azure_mgmt_service_bus/spec/namespaces_spec.rb

```diff
@@ -17,18 +17,18 @@
 
   it 'should create list and delete namespace' do
     namespace_name = "smaple34566"
-    namespace_parameters = Azure::ARM::ServiceBus::Models::NamespaceCreateOrUpdateParameters.new
+    namespace_parameters = Azure::ARM::ServiceBus::Models::SBNamespace.new
```

Comment:
similarly for this gem, since we know of these type of changes, we should write something down in our change log / release notes.